shared/modules/add-nonce-to-csp.ts

```diff
@@ -0,0 +1,38 @@
+// ASCII whitespace is U+0009 TAB, U+000A LF, U+000C FF, U+000D CR, or U+0020 SPACE.
```

Comment:
Wow. I love how simple this ended up being in the end! Quite the journey you went on to get here! Great work!